reactive-banana/src/Reactive/Banana/Prim/Graph.hs

```diff
@@ -5,7 +5,14 @@
 ------------------------------------------------------------------------------}
 {-# language ScopedTypeVariables#-}
 
-module Reactive.Banana.Prim.Graph where
+module Reactive.Banana.Prim.Graph
```

Comment:
To make sure some invariants are not violated, made `Graph` an opaque type to the rest of the codebase.